examples/MeteorCLI/all-in-one/imports/app/app.module.ts

```diff
@@ -27,7 +28,7 @@ import { PageNotFoundComponent } from './page-not-found/page-not-found.component
       },
       {
         path: 'todoAdd',
-        loadChildren: './todo-add/todo-add.module#TodoAddModule',

```

Comment:
This line is replaced by compiler with Meteor's dynamic loading syntax, so if you add it regular way, this isn't going to be loaded lazily.

Reply:
Thanks @ardatan!
I must have missed it. There were some initial errors, while still upgrading to a newer Angular Meteor compiler version.